src/array/indexOf.js

```diff
@@ -0,0 +1,29 @@
+const indexOf = (searchArray, searchValue, searchAt) => {
+    if (searchArray.length - 1 < searchAt) {
+        return -1;
+    } else if (
+        searchArray.includes(searchValue) === false ||
```

Comment:
Instead of doing all of these checks, I would rather pass in defaults into the params of the function and then write logic that accounts for what the passed in values are.